### ISSUE TITLE
where did you use the joint loss?

### ISSUE THREAD

**Issue by micklexqg** [neka-nat/image_completion_keras#1]
thanks for your work. I am running your code and have some questions.
where did you use the joint loss in your code? And I notice the weight for discrimitor loss is 0.0004 in the original paper. where did you set the parameter?
thanks, any help appreciated!

**Comment by neka-nat** [comment]
Thanks @micklexqg .
My code is referring to [glcic](https://github.com/tadax/glcic) and their implementation seems to have [a similar issue](https://github.com/tadax/glcic/issues/2).
I made the new branch implemented according to the paper, so please check it.
https://github.com/neka-nat/image_completion_keras/pull/2

**Comment by micklexqg** [comment]
thanks for quich reply. i checked it, but maybe there is still no update for joint loss...

**Comment by neka-nat** [comment]
I updated again.
https://github.com/neka-nat/image_completion_keras/pull/2/files#diff-e44f4a60e89e820dde9bb27afa634965R135

**Comment by micklexqg** [comment]
@neka-nat , thank you, need i copy the code from the given url?(download from the github clone is still the old version ). another question, is there any pretrained model, I think uploading a pretrained model is better.

**Comment by neka-nat** [comment]
If you use new version branch, please run the follow commands.
```
git clone https://github.com/neka-nat/image_completion_keras.git
cd image_completion_keras
git reset origin/separate_c_d --hard
```
Sorry, I can not prepare the pre-train model yet.
Please wait several weeks.

**Comment by micklexqg** [comment]
@neka-nat ,thanks, if it is finished ,please inform us. thanks again!
